Assets/Prefabs/Manager/ManagerGameObject/Manager.cs

```diff
@@ -277,8 +282,13 @@ public void Update()
     {
         if (started)
         {
-
             moneyText.text = virtualCurrency.SealsCurrency.ToString();
+            if (Input.GetKeyDown(KeyCode.F5))
+            {
+                virtualCurrency.AddCurrency();
+            }

```

Comment:
Braces not necessary, can just single line this

Reply:
Good point